.travis.yml

```diff
@@ -27,6 +27,7 @@ before_script:
   - isort --check-only --diff
   - python -m django check --settings=tests.settings
   - python -m django makemigrations --settings=tests.settings --no-input --dry-run --check
+  - python -m django makemigrations test_project --settings=tests.test_project.settings
```

Comment:
This feels a bit odd to me on the surface. Could this instead be a part of running the test? I'd like to avoid an extra test setup step if possible so that the tests are immediately runnable as before.

Reply:
ok I'll see what I can do, I just wanted to avoid executing a script within the tests. but maybe I can just use `call_command()`, will try.